### ISSUE TITLE
OH DC Updates

### ISSUE THREAD

**Issue by sunicolita** [sciencehistory/scihist_digicoll#1553]
During our last Center for Oral History meeting, our team discussed a couple of updates to the OH Collection pages that we would be interested in looking into with @eddierubeiz and @jrochkind as their schedules permit.

1.   As we pursue reparative description efforts, we have considered individually editing transcripts to include a brief description of our editorial decisions. Instead of having to update 700+ oral history transcripts manually, can we find a space on the Oral History Collections landing page for these editorial policies and disclaimers? Is this something that could be easily done?

2.   We're interested in reaching out to interviewees for updated contact and demographics information and are thinking ahead to the longevity and organization of the kinds of forms that we would ask interviewees to fill out for us. What are your thoughts on having automatic submission forms set up like the Request Access to oral histories button on the aforementioned Editorial Decisions tab? I understand that the DC may not be the place for this, but am interested in additional thoughts. This is a small ticket item that would be nice to have online but is not a make-or-break update.

3.   We discovered errors in the citation generator when it comes to names and dates. I was made aware of the long-standing issue with the dates [sciencehistory/scihist_digicoll/issues/574](https://github.com/sciencehistory/scihist_digicoll/issues/574) that go back a couple of years, but I don't understand where the ticket was left off. An explanation of how the generator pulls metadata from the backend would be helpful to the COH so we can discuss any adjustments we can make ourselves to help improve the accuracy of the generator. 

Thanks for your help!

**Comment by jrochkind** [comment]
Thanks! This sounds like three different issues! Ideally it might be three different tickets, as they aren't related to each other at all, just three different things, I think?

i don't totally understand the first two. It might be a few weeks until we have time to talk about this, but at that point maybe we'll set up a meeting to discuss these issues?

**Comment by sunicolita** [comment]
Sounds like a plan - thanks! I grouped them together as general updates, but should I edit this thread and submit 3 different tickets instead? (sorry - not fluent in GitHub best practices yet!)

**Comment by jrochkind** [comment]
No worries for now, we can separate them as we get to them. In the future, yeah, I'd say what works best is if something can be divided into independent unrelated things, make them separate tickets!

**Comment by apinkney0696** [comment]
@sunicolita and I met this morning to chat about DC and OH workflows and how I will take over some oral history work in the digital collections in the interim prior to her replacement. We chatted about this ticket and determined the following solutions/decisions:

1. Transcripts will be updated individually with an additional page containing a brief description of editorial decisions. If there comes a point where there are a bunch of transcript files to be updated in the digital collections, I may put in a ticket to automate, but this may not be necessary as transcripts will be updated gradually.

2. This is a workflow issue. The COH is seeking a form to use while communicating with interviewees. I introduced the possibility of using the support portal for this purpose rather than the digital collections. This seems to be a solution that will better fit the needs of the COH.

3. Determined that the errors are negligible. The disclaimer beneath the citation is sufficient for now. 

I'm happy to talk through more of these decisions and the context of the original ticket points at our next meeting now that I have a better idea from Nic of what they were originally asking for. I'm going to close this ticket for now.
